Shop.API/appsettings.json

```diff
@@ -7,3 +7,4 @@
   },
   "AllowedHosts": "*"
 }
+0
```

Comment:
what is it?

Reply:
It's a prank there's a camera over there🥲

Reply:
Solved
My bad